modules/op-deploying-ui-for-tekton-hub.adoc

```diff
@@ -0,0 +1,89 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="deploying-the-ui-for-tekton-hub_{context}"]
+= Deploying the user interface (UI) for {tekton-hub}
+
+[discrete]
+.Prerequisites
+* Ensure that you are in the `<project_root>/config/` directory, where `<project_root>` is the top level directory of the cloned {tekton-hub} repository.
+
+[discrete]
+.Procedure
+. Set up the route or ingress.
++
+[source,terminal]
+----
+$ oc apply -f 04-openshift/41-ui-route.yaml -n tekton-hub
+----
+
+. Update the `ConfigMap` object for the UI.
+.. Note the URL of the {tekton-hub} API by running the following command:
++ 
+[source,terminal]
+----
+$ oc get -n openshift-pipelines routes tekton-hub-api --template='https://{{ .spec.host }}'
+----
+.. Edit the `03-ui/30-ui-configmap.yaml` file to set the URL for API, authorization, and UI redirection.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tekton-hub-ui
+data:
+  API_URL: <API_URL>
+  API_VERSION: <API_version>  <<< Update this by API version (For e.g-"v1")
+  AUTH_BASE_URL: <auth_URL>
+  REDIRECT_URI: <UI_URL>
+----
+
+. Update the UI image.
+.. Edit the `03-ui/30-ui-configmap.yaml` file to include the URL of the UI image in the image registry.
++
+[source,yaml]
+----
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tekton-hub-ui
+  ...
+spec:
+  ...
+    spec:
+      containers:
+        - name: tekton-hub-ui
+          image: quay.io/tekton-hub/ui <1>
+          ...
+----
+<1> Provide the URL of the UI image in the image registry.
+.. Deploy the UI.
++
+[source,terminal]
+----
+$ oc apply -f 03-ui/ -n tekton-hub
+----
+.. Ensure that the pods are running.
++
+[source,terminal]
+----
+$ oc get pods -n openshift-pipelines
+
+NAME                   READY   STATUS      RESTARTS   AGE
+tekton-hub-api-6dfc6f97d9-vk66r   1/1     Running     3          21m
+tekton-hub-db-9bd4cdf99-zsq89     1/1     Running     0          21m
+tekton-hub-db-migration-26ngs     0/1     Completed   0          21m
+tekton-hub-ui-55fc66cc6b-69dsp    1/1     Running     0          58s
+----
+.. Verify that you can access the UI route by opening the URL returned by the following command:
++
+[source,terminal]
+----
+$ oc get routes -n openshift-pipelines tekton-hub-ui --template='https://{{ .spec.host }} '
+----
++
+The UI does not display any resources as the database is empty, but the **Categories** on the left panel should be visible.
```

Comment:
```suggestion
The UI does not display any resources because the database is empty, but the **Categories** on the left panel should be visible.
```
Use "because" instead of "as" to indicate causation.

Reply:
See IBM SG: 
user interface, UI noun
Use these terms if you know that your intended audience is familiar and comfortable with what they
represent, but in most cases you can write instructions that tell how to access the user interface without using a generic noun.
Avoid "in the UI, do this" or "in the interface, do that". In tasks, refer to the specific part of the user interface,
such as the name of a window, field, pane, or view. Use "UI" or "user interface" when you need to contrast
with named non-UI components or character-based interfaces, such as the DB2 command-line interface.